### PR TITLE
DOCS-1365 New RUM page: tracking user actions

### PR DESCRIPTION
**Pull request by kayayarai**
### What does this PR do?
Adds a new docs page for tracking user actions in RUM.

### Motivation
DOCS-1365

### Preview
https://docs-staging.datadoghq.com/kari/docs-1365-rum-user-actions/real_user_monitoring/browser/tracking_user_actions/

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by hdelaby — APPROVED** [review]

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ These performance metrics are key for helping you start your investigations:
 
 ## All performance metrics
 
-For information about the default attributes for all RUM event types, see [Data Collected][5]. For information about configuring for sampling, global context, or custom user actions, see [Advanced Configuration][6]. The following table lists Datadog-specific metrics along with performance metrics collected from [Navigation Timing API][7] and [Paint Timing API][8]:
+For information about the default attributes for all RUM event types, see [Data Collected][5]. For information about configuring for sampling or global context see [Advanced Configuration][6]. The following table lists Datadog-specific metrics along with performance metrics collected from [Navigation Timing API][7] and [Paint Timing API][8]:
```
Comment (cswatt) [inline content/en/real_user_monitoring/browser/monitoring_page_performance.md:42]:
```suggestion
For information about the default attributes for all RUM event types, see [Data Collected][5]. For information about configuring for sampling or global context see [Advanced Configuration][6]. The following table lists Datadog-specific metrics along with performance metrics collected from the [Navigation Timing API][7] and [Paint Timing API][8]:
```

File: content/en/real_user_monitoring/browser/tracking_user_actions.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+    text: "Build analytics upon your events"
+  - link: "/real_user_monitoring/dashboards/"
+    tag: "Documentation"
+    text: "RUM Dashboards"
+---
+
+
+Real User Monitoring (RUM) SDKs detect user interactions performed during a user journey. Set the `trackInteractions` [initialization parameter][1] to `true` to enable this feature.
+
+**Note**:  The `trackInteractions` initialization parameter enables the collection of user clicks in your application. **Sensitive and private data** contained on your pages may be included to identify the elements interacted with.
```
Comment (cswatt) [inline content/en/real_user_monitoring/browser/tracking_user_actions.md:25]:
```suggestion
**Note**: The `trackInteractions` initialization parameter enables the collection of user clicks in your application. **Sensitive and private data** contained on your pages may be included to identify the elements interacted with.
```

File: content/en/real_user_monitoring/browser/tracking_user_actions.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+Once an interaction is detected, all new RUM events are attached to the ongoing action until it is considered finished. The action also benefits from its parent view attributes such as browser information, geolocation data, [global context][2].
```
Comment (cswatt) [inline content/en/real_user_monitoring/browser/tracking_user_actions.md:27]:
```suggestion
Once an interaction is detected, all new RUM events are attached to the ongoing action until it is considered finished. The action also benefits from its parent view attributes such as browser information, geolocation data, and [global context][2].
```

File: content/en/real_user_monitoring/browser/tracking_user_actions.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+| Metric    | Type   | Description              |
+|--------------|--------|--------------------------|
+| `action.loading_time` | number (ns) | The loading time of the action.  |
+| `action.long_task.count`        | number      | Count of all long tasks collected for this action. |
+| `action.resource.count`         | number      | Count of all resources collected for this action. |
+| `action.error.count`      | number      | Count of all errors collected for this action.|
+
+### How action loading time is calculated
+
+Once an interaction is detected, the RUM SDK watches for network requests an DOM mutations. It is considered finished once the page has no activity for more than 100ms (activity being defined as ongoing network requests or DOM mutations).
```
Comment (cswatt) [inline content/en/real_user_monitoring/browser/tracking_user_actions.md:42]:
```suggestion
Once an interaction is detected, the RUM SDK watches for network requests and DOM mutations. It is considered finished once the page has no activity for more than 100ms (activity being defined as ongoing network requests or DOM mutations).
```

File: content/en/real_user_monitoring/browser/tracking_user_actions.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+
+## Action attributes
+
+| Attribute    | Type   | Description              |
+|--------------|--------|--------------------------|
+| `action.id` | string | UUID of the user action. |
+| `action.type` | string | Type of the user action. For custom user actions, it is set to `custom`. |
+| `action.target.name` | string | Element that the user interacted with. Only for automatically collected actions |
```
Comment (cswatt) [inline content/en/real_user_monitoring/browser/tracking_user_actions.md:50]:
```suggestion
| `action.target.name` | string | Element that the user interacted with. Only for automatically collected actions. |
```

File: content/en/real_user_monitoring/browser/tracking_user_actions.md [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+
+<div class="alert alert-danger" role="alert" data-dd-action-name="Dismiss alert">
+  <span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span>
+  <span class="sr-only">Error:</span>
+  Enter a valid email address
+</div>
+```
+## Custom user actions
+
+Custom user actions are user actions declared and sent manually by using the `addUserAction` API. They can send information relative to an event occurring during a user journey, for example, a custom timing or customer cart information.
```
Comment (cswatt) [inline content/en/real_user_monitoring/browser/tracking_user_actions.md:70]:
```suggestion
Custom user actions are user actions declared and sent manually by using the `addUserAction` API. They can send information relative to an event occurring during a user journey (for example, a custom timing or customer cart information).
```

File: content/en/real_user_monitoring/browser/tracking_user_actions.md [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+
+Once RUM is initialized, generate user actions when you want to monitor specific interactions on your application pages or measure custom timings with the `addUserAction(name: string, context: Context)` API. In the following examples, the RUM SDK collects the amount of items within a cart, what they are, and how much the cart is worth overall.
```
Comment (cswatt) [inline content/en/real_user_monitoring/browser/tracking_user_actions.md:72]:
```suggestion
Once RUM is initialized, generate user actions when you want to monitor specific interactions on your application pages or measure custom timings with the `addUserAction(name: string, context: Context)` API call. In the following examples, the RUM SDK collects the number of items within a cart, what they are, and how much the cart is worth overall.
```

**Review by cswatt — CHANGES_REQUESTED** [review]
editorial review

**Review by cswatt — APPROVED** [review]

**Pull request merged**